src/main/webapp/app/shared/model/firebase/firebase.model.ts

```diff
@@ -234,6 +236,17 @@ export class Mutation {
   }
 }
 
+export type Flag = {
+  type: string;
+  flag: string;
```

Comment:
I think we should pull the flags from the server side when loading the page. Andonly keep the type and flag here.